VideoLocker/res/layout/header_discussion_topics_list.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/white"
+    android:orientation="vertical">
+
+    <org.edx.mobile.view.custom.ETextView
+        android:id="@+id/discussion_topic_all_posts"
+        style="@style/regular_grey_text"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="?android:attr/selectableItemBackground"
+        android:ellipsize="end"
+        android:gravity="center_vertical"
+        android:padding="@dimen/edx_margin"
+        android:singleLine="true"
+        android:text="@string/discussion_posts_filter_all_posts"
+        android:textDirection="locale"
+        android:textSize="@dimen/edx_base" />
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/edx_hairline"
+        android:background="@color/edx_grayscale_neutral_x_light" />
```

Comment:
This separator does not have the same color and height as the `ListView` divider. It might be worth it to separate this layout into two headers, that we can make use of the existing `listSelector` and `divider`. Note that in that case you'll need to have the header enabled, and handle the click from the `OnItemClickListener` of the `ListView`.